frontend/src/browser-router.js

```diff
@@ -0,0 +1,14 @@
+import { createBrowserRouter } from "react-router-dom";
+
+import TextField from '@mui/material/TextField';
+
+const Input = createBrowserRouter([
```

Comment:
Please change the name, in future it will contain much more than just an input field. This is your apps main router